ipwb/replay.py

```diff
@@ -787,12 +785,17 @@ def generateNoMementosInterface_noDatetime(urir):
     return msg
 
 
+@app.errorhandler(404)
+def page_not_found(e):
+    return "<h1>ERROR 404</h1>Not Found", 404
```

Comment:
* Wrapped in paragraph tag
* Sentence-case for consistency

```suggestion
    return "<h1>ERROR 404</h1><p>Resource not found</p>", 404
```